src/Sarif.Multitool.FunctionalTests/Rules/EndColumnMustNotBeLessThanStartColumnTests.cs

```diff
@@ -50,10 +37,10 @@ public void EndColumnMustNotBeLessThanStartColumn_EndColumnLessThanStartColumnIn
             Verify(new EndColumnMustNotBeLessThanStartColumn(), "EndColumnLessThanStartColumnInRelatedLocation.sarif");
         }
 
-        [Fact(DisplayName = nameof(EndColumnMustNotBeLessThanStartColumn_EndColumnLessThanStartColumnInResultFile))]
-        public void EndColumnMustNotBeLessThanStartColumn_EndColumnLessThanStartColumnInResultFile()
+        [Fact(DisplayName = nameof(EndColumnMustNotBeLessThanStartColumn_EndColumnLessThanStartColumnInResultLocation))]
```

Comment:
Changed the name of this test because there's no longer such a thing as a "result file" -- there are just `location` objects on a `result`.